build/_build.csproj

```diff
@@ -15,16 +15,17 @@
         <Description>Cross-platform  build automation system for VirtoCommerce projects</Description>
         <NeutralLanguage>en-US</NeutralLanguage>
         <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
-        <PackageVersion>3.0.0-beta0005</PackageVersion>
+        <PackageVersion>3.0.0-beta0008</PackageVersion>
+        <StartupObject>Build</StartupObject>
     </PropertyGroup>
 
-    <ItemGroup>      
-        <PackageReference Include="GitVersion.CommandLine" Version="5.1.3">
+    <ItemGroup>
+        <PackageReference Include="GitVersion.CommandLine" Version="5.1.2">
           <PrivateAssets>all</PrivateAssets>
           <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-        </PackageReference>      
+        </PackageReference>
         <PackageReference Include="Nuke.Common" Version="0.23.6" />
-        <PackageReference Include="VirtoCommerce.Platform.Core" Version="3.0.0-beta0004" />
+        <PackageReference Include="VirtoCommerce.Platform.Core" Version="3.0.0-rc3-0141" />
```

Comment:
Better to make there project reference to prevent copying obsolete assemblies to publish